invenio_communities/assets/semantic-ui/js/invenio_communities/community/frontpage.js

```diff
@@ -123,7 +123,7 @@ class CommunitiesCardGroup extends Component {
         {this.state.data.hits.length === 0 ? (
           <EmptyMessage message={this.props.emptyMessage} />
         ) : (
-          <Card.Group itemsPerRow={5} className="community-frontpage-cards">
+          <Card.Group doubling stackable itemsPerRow={5} className="community-frontpage-cards">
```

Comment:
I think the classname can be removed, I don't remember seeing it in the CSS, to be checked